cmd/influxd/launcher/launcher_helpers.go

```diff
@@ -367,10 +367,6 @@ func (tl *TestLauncher) BucketService(tb testing.TB) *http.BucketService {
 	return &http.BucketService{Client: tl.HTTPClient(tb)}
 }
 
-func (tl *TestLauncher) CheckService() influxdb.CheckService {
```

Comment:
Note for the reader:

You'll see this have moved up onto the Launcher for now. I have been moving more of these helpers to use the clients as opposed to in-process service implementations. I went to move this to the `http.CheckService` implementation. But half the methods don't actually match the `influxdb.CheckService` interface. They return their own local check API domain model and accept slightly different arguments.
I started down the path of fixing this client, but It started to get unwieldy. I still want to deliver that though as its own PR later. So I have made a node to create and issue and do that.
Until then this just updates the launcher test suite to interface with the same check service installed within the launcher itself. So that consistency is at least achieved, even if it doesn't exercise the front door as we would like.